railties/lib/rails/generators/rails/scaffold_controller/USAGE

```diff
@@ -12,7 +12,7 @@ Description:
 Example:
     `rails generate scaffold_controller CreditCard`
 
-    Credit card controller with URLs like /credit_card/debit.
```

Comment:
Yea I agree with @kaspth here. Can you fix the url? I don't think the new phrasing is clearer, nor do I think listing out 7 URLs is valuable.